authors/settings.py

```diff
@@ -80,8 +82,11 @@
 
 DATABASES = {
     'default': {
-        'ENGINE': 'django.db.backends.sqlite3',
-        'NAME': os.path.join(BASE_DIR, 'db.sqlite3'),
+        'ENGINE': 'django.db.backends.postgresql',
+        'NAME': os.getenv("DB_NAME"),
+        'USER': os.getenv("DB_USER"),
```

Comment:
Also add default if env variable is not set. `os.getenv("ENV_VAR", <DEFAULT_VALUE>)`.

Reply:
oh alright. Working on it